### PR TITLE
fix invalid test session

### PR DESCRIPTION
**Pull request by guangxuli**
fix these:
1. make test invalid
2. example pkg/build no test files

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -55,11 +55,11 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+test check:
```
Comment (bparees) [inline Makefile]: the "check" target should run the build, verify, and test targets.
the "test" target should run the hack/test-go logic.

Comment (guangxuli) [inline Makefile]: oh, i added this according to the comments 
```
#   check: Run tests.
#   test: Run tests.
```

agree, fixed. origin also did like what your specified. thanks.

File: Makefile [hunk 2]
```diff
@@ -55,12 +55,21 @@ install-travis:
 # Example:
 #   make check
 #   make test
-#   make check WHAT=pkg/build TESTFLAGS=-v
-check:
+#   make check WHAT=pkg/docker TESTFLAGS=-v
+check:	verify
```
Comment (bparees) [inline Makefile]: why not just `check: verify test` ?

and move verify-bash-completion to the verify target above.

Comment (guangxuli) [inline Makefile]: because verify have include build.

Comment (bparees) [inline Makefile]: but what is the difference between calling $(MAKE) test and having test as a dependency of the check target?

Comment (bparees) [inline Makefile]: i don't think there is one, since you're not passing any arguments, thus there's no reason to do it that way.

Comment (guangxuli) [inline Makefile]: yes, I should use test target instead of $(MAKE) test after moving verify-bash-completion script.

**Comment by guangxuli** [comment]
@bparees any more comments? thanks!

**Comment by bparees** [comment]
lgtm
[merge]

**Pull request merged**